spec.md

```diff
@@ -4070,7 +4076,7 @@ not have any values in common, and `False` otherwise.
 
 This is equivalent to `not S.intersection(x)`.
 
-`isdisjoint` fails if `x` is not an iterable sequence.
```

Comment:
Did you drop the "not an iterable sequence" wording because it's implied/obvious from context?

Reply:
Yes - the first line already states that `x` must be an iterable sequence.